src/main/java/com/stuypulse/robot/commands/swerve/SwerveDriveBalanceAlign.java

```diff
@@ -0,0 +1,30 @@
+package com.stuypulse.robot.commands.swerve;
+
+import com.stuypulse.robot.subsystems.odometry.Odometry;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class SwerveDriveBalanceAlign extends SwerveDriveToPose {
+    public SwerveDriveBalanceAlign() {
+        super(() -> new Pose2d(Odometry.getInstance().getTranslation(), new Rotation2d(getAngle())));
```

Comment:
use () -> { } so you can cleanly store odometry and then pass in all the necessary things to the helper methods